source/common/common/thread.h

```diff
@@ -6,6 +6,10 @@
 
 #include "envoy/thread/thread.h"
 
+#include "common/common/non_copyable.h"
```

Comment:
I think these includes can be removed now?

Reply:
+1